<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>885</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1154</width>
    <height>885</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1154</width>
    <height>885</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>道路违法检测系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>0</y>
      <width>51</width>
      <height>831</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="DisplayLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>501</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>711</width>
      <height>431</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>801</width>
      <height>501</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>                                                  视频播放区域</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>411</width>
      <height>291</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_road">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>监控路口：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_road_text">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请先设置路口名称！</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-25</y>
      <width>1171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>100</y>
      <width>341</width>
      <height>251</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>311</width>
      <height>241</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>10000</width>
      <height>10000</height>
     </size>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>810</y>
      <width>1171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>801</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Button_GetPos">
       <property name="text">
        <string>参数配置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Continue">
       <property name="text">
        <string>暂停/继续</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Close">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Api_button">
       <property name="text">
        <string>检测</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Open">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>380</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>71</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>141</width>
      <height>71</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>导入待检测视频文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_imgout">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>500</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>71</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>141</width>
      <height>71</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查看违章帧</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_infoout">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>620</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>71</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>141</width>
      <height>71</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>导出违章文本信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_quit">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>740</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>71</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>141</width>
      <height>71</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出系统</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_light">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>时间段：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_date_text">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>50</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>29</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>807</x>
      <y>79</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_roadinfo">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>540</y>
      <width>391</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>                   车流量信息展示区域</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1154</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="action_SetRoad"/>
    <addaction name="separator"/>
    <addaction name="action_choose_road"/>
    <addaction name="separator"/>
    <addaction name="delete_cache"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_helper">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_helper"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDelet_existdata">
   <property name="text">
    <string>清除数据（包括违法记录）</string>
   </property>
  </action>
  <action name="delete_cache">
   <property name="text">
    <string>清理缓存</string>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>查看使用手册</string>
   </property>
  </action>
  <action name="actionZidonghua">
   <property name="text">
    <string>自动化检测</string>
   </property>
  </action>
  <action name="action_choose_road">
   <property name="text">
    <string>查看已处理视频</string>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_SetRoad">
   <property name="text">
    <string>设置当前路口</string>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Button_quit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>808</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>809</x>
     <y>707</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
